VideoWeb/VideoWeb.UnitTests/Controllers/ConfigSettingController/GetClientConfigurationSettingsTests.cs

```diff
@@ -65,13 +72,31 @@ public void Should_return_response_with_settings()
             JoinByPhoneFromDate = "2021-02-09"
         };
         
-        var supplierLocatorMock = _mocker.Mock<ISupplierLocator>()
-            .Setup(x => x.GetSupplierConfiguration())
-            .Returns(Options.Create(kinlyConfiguration));
+        var vodafoneConfiguration = new VodafoneConfiguration
+        {
+            JoinByPhoneFromDate = "2022-02-09"
+        };
+
+        var supplierConfigurations = new List<SupplierConfiguration>
+        {
+            kinlyConfiguration
+        };
+        if (vodafoneEnabled)
```

Comment:
if statements in setup makes for a confusing setup. I thin we should have a test for true and false with a single flow rather than conditional ones

Reply:
split into separate tests